ashley/src/main/kotlin/ktx/ashley/listeners.kt

```diff
@@ -0,0 +1,124 @@
+package ktx.ashley
+
+import com.badlogic.ashley.core.Engine
+import com.badlogic.ashley.core.Entity
+import com.badlogic.ashley.core.EntityListener
+import com.badlogic.ashley.core.Family
+import com.badlogic.ashley.systems.IntervalIteratingSystem
+import com.badlogic.ashley.systems.IteratingSystem
+import com.badlogic.ashley.systems.SortedIteratingSystem
+
+/**
+ * An [EntityListener] only listening to the [entityAdded] event.
+ *
+ * An empty implementation is provided for the [entityRemoved] event, as it is ignored.
+ *
+ * If you need both events from the [EntityListener], implement the regular [EntityListener] from Ashley.
+ */
+interface EntityAdditionListener : EntityListener {
+  override fun entityAdded(entity: Entity)
```

Comment:
The only thing that comes to mind at this point is that we might want to include some documentation for the methods that should be overridden, i.e. something along the lines of `Invoked when an matching [Entity] is added to the engine. See [EntityListener.entityAdded].`.

Reply:
Resolved in https://github.com/libktx/ktx/pull/410/commits/56811de1b8d027612a7422f7a36cc53205a68cf5.